packages/docs/src/page-configs/ui-elements/backtop/api-options.ts

```diff
@@ -1,11 +1,6 @@
 import { defineManualApi } from '../../../components/DocsApi/ManualApiOptions'
 
 export default defineManualApi({
-  events: {
-    click: {
-      types: 'Event',
```

Comment:
`VaBacktop` has use this event. So there are some event didn't emitted, but are used in demo. What kind of event have this characteristic, currently I am looking at relevant files such as `VaBacktop.vue` and `VaBacktop.demo.vue`, but what if event don't have example in `*.demo.vue`?

Reply:
Same. It is DOM event, not vue emit.